### PR TITLE
ghost: add page

### PR DESCRIPTION
**Pull request by stets**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ghost.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`ghost restart`
+
+- Check the system for any potential hiccups when on install or update of Ghost:
+
+`ghost doctor`
+
+- View the logs of a Ghost instance:
+
+`ghost log [name]`
```
Comment (zdroid) [inline pages/common/ghost.md:24]: This is confusing. I guess `[name]` should be `{{name}}` here, as that's the syntax we use for variable parameters.

File: pages/common/ghost.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`ghost run`
+
+- View running ghost processes:
+
+`ghost ls`
+
+- View or edit Ghost configuration:
+
+`ghost config [key] [value]`
```
Comment (zdroid) [inline pages/common/ghost.md:36]:
```suggestion
`ghost config [key] [value]`
```
Same as above. Although I wouldn't mesh up both viewing and editing the configuration, it's confusing.

Comment (sbrl) [reply pages/common/ghost.md:36]: Do you mean this @zdroid?
```suggestion
`ghost config {{key}} {{value}}`
```

File: pages/common/ghost.md (hunk 1 above)
Comment (stets) [reply pages/common/ghost.md]: Thanks, added this one

**Pull request merged**